admin/server.go

```diff
@@ -28,6 +28,9 @@ func (s *Server) ListenAndServe(addr string) error {
 	http.Handle("/routes", &ui.RoutesHandler{Color: s.Color, Title: s.Title, Version: s.Version})
 	http.HandleFunc("/health", handleHealth)
 	http.Handle("/", http.RedirectHandler("/routes", http.StatusSeeOther))
+	if s.Cfg.UI.CertFile != "" {
```

Comment:
OK, I've just written that from memory from the time I've coded the cert stores. I'll have another look.